code/integration-tests/src/androidTest/java/com/adobe/marketing/mobile/integration/identity/IdentityIntegrationTests.kt

```diff
@@ -310,28 +311,49 @@ class IdentityIntegrationTests {
         countDownLatch.await()
     }
 
+    @Test(timeout = 5000)
     @Ignore
-    @Test(timeout = 10000)
-    //TODO: enable this test once MobileCore.getSdkIdentities event handling is implemented in the new Configuration extension
+    //TODO: Fix issue where Analytics does not update persistence and shared state with latest vid
     fun testGetSdkIdentities() {
-        val countDownLatch = CountDownLatch(1)
         MobileCore.updateConfiguration(
             mapOf(
                 "experienceCloud.org" to "orgid",
                 "experienceCloud.server" to "test.com",
                 "global.privacy" to "optedin"
             )
         )
+
+        Analytics.setVisitorIdentifier("testVid")
+        val analyticsLatch = CountDownLatch(1)
+        Analytics.getVisitorIdentifier {
+            analyticsLatch.countDown()
+        }
         MobileCore.setAdvertisingIdentifier("adid")
         Identity.syncIdentifiers(mapOf("id1" to "value1"))
+        val identityLatch = CountDownLatch(1)
+        var ecid : String? = null
+        Identity.getExperienceCloudId{
+            ecid = it
```

Comment:
Add an assert that `ecid` is not null. You can add this at the end with the other asserts.